server/src/main/java/swim/tutorial/UnitAgent.java

```diff
@@ -12,14 +12,36 @@
 import java.util.Iterator;
 
 public class UnitAgent extends AbstractAgent {
+		
+  // TODO: complete the stats Value Lane
+  // @SwimLane("stats")
+	
+  // HINT: Use the valueLane() method to instantiate the lane
+  // HINT: Use the .didSet() lifecycle callback to log a message showing updates to stats
+	
+   @SwimLane("histogram")
+   private final MapLane<Long, Value> histogram = this.<Long, Value>mapLane()
+       .didUpdate((k, n, o) -> {
+         logMessage("histogram: replaced " + k + "'s value to " + Recon.toString(n) + " from " + Recon.toString(o));
+         
+         // TODO: update stats with update logic
+         
+         // HINT: access new data sent to histogram with 
```

Comment:
Let's replace this with `n.get("count").longValue()` for now.

Reply:
Not sure if the comments from (41-48) get too overwhelming. If so, I can remove the `#RECOMMENDED` and `#ALTERNATIVELY` sections and just leave it at suggesting `n.get("count).longValue()`

Reply:
I actually like the RECOMMENDED and ALTERNATIVELY!